Lib/test/test_generated_cases.py

```diff
@@ -4,6 +4,7 @@
 import sys
 import tempfile
 import unittest
+import textwrap
```

Comment:
Nit: retain alphabetical order:
```suggestion
import tempfile
import textwrap
import unittest
```